lib/modules/smart/test/shared_launch_tests_test.rb

```diff
@@ -123,9 +123,11 @@ class SharedLaunchTestSequence < Inferno::Sequence::SequenceBase
     before do
       @test = @sequence_class[:invalid_code]
       @sequence = @sequence_class.new(@instance, @client)
-      @instance.redirect_uris = 'http://www.example.com/redirect'
-      @instance.client_id = 'CLIENT_ID'
-      @instance.oauth_token_endpoint = 'http://www.example.com/token'
+      @instance.update!(
```

Comment:
It depends on whether the values need to be persisted to the database. `@instance.whatever = something` doesn't persist anything. I'd generally rather do
```
@instance.update!(whatever: something)
```
than
```
@instance.whatever = something
@instance.save!
```
If there's a chance that the model is dirty, however, using `update` is a bad idea because it will only persist the fields in the `update` call, so the other changes will be lost.